crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/initiate.rs

```diff
@@ -67,6 +68,7 @@ impl ShardedGossipLocal {
                 inner.initiate_tgt = Some(tgt);
                 Ok(())
             })?;
+            // TODO always a new connection even if we have an open one?
```

Comment:
Missed this previously, if we gossip with somebody more than once then shouldn't we be re-using the connection if we can?

Reply:
Ahh right! I'll get rid of this TODO then. Thank you